backend/web/src/frontend/src/store/index.js

```diff
@@ -1,11 +1,37 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
+import BookmarkService from '../api/module/bookmark';
+import { FETCH_BOOKMARKS } from './share/actionType';
+import { SET_BOOKMARKS } from './share/mutationsType';
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
-  state: {},
-  mutations: {},
-  actions: {},
-  modules: {},
+  state: {
+    bookmarks: [
+      {
+        id: '',
+        url: '',
+        tags: [{ id: '3', name: 'jordy' }],
+      },
+    ],
+  },
+  getters: {
+    bookmarks(state) {
+      return state.bookmarks;
+    },
+  },
+  mutations: {
+    [SET_BOOKMARKS](state, bookmarks) {
+      state.bookmarks = bookmarks;
+    },
+  },
+  actions: {
+    async [FETCH_BOOKMARKS]({ commit }) {
+      return BookmarkService.getAll().then(({ data }) => {
+        commit(SET_BOOKMARKS, data);
+        return data;
+      });
+    },
```

Comment:
```suggestion
    async [FETCH_BOOKMARKS]({ commit }) {
      const bookmarks = await BookmarkService.getAll();
      commit(SET_BOOKMARKS, bookmarks);
      return bookmarks;
    },
```
다음과 같은 형태가 가독성 측면에서 좋을 것 같네요.